shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -421,14 +421,16 @@ final class ImageReaderSurfaceProducer
     // If we cleanup the ImageReaders on memory pressure it breaks VirtualDisplay
```

Comment:
Did you want to update this comment since the issue was not samsung specific?

Reply:
100 should be